docs/api/spec.md

```diff
@@ -78,29 +78,31 @@ Examples of End Points include:
 ### Open an Application by Name
 Linking from one application to another is a critical basic workflow that the web revolutionized via the hyperlink.  Supporting semantic addressing of applications across different technologies and platform domains greatly reduces friction in linking different applications into a single workflow.
 
-### Raising Intents
+### Raising Intents or Contexts
 Often, we want to link from one app to another to dynamically create a workflow.  Enabling this without requiring prior knowledge between apps is a key goal of FDC3.
 
-Intents provide a way for an app to request functionality from another app and defer the discovery and launching of the destination app to the Desktop Agent.  There are multiple models for interop that Intents can support.
+Intents provide a way for an app to request functionality from another app and defer the discovery and launching of the destination app to the Desktop Agent.  There are multiple models for interop that Contexts and Intents can support.
```

Comment:
Here as well, this paragraph really refers to Intents, not to Contexts (which can't be raised in and of themselves) - just sticking to Intents as the thing that can be raised will make this clearer.